.github/workflows/build-images-base.yaml

```diff
@@ -27,6 +27,10 @@ on:
         description: WASM Shim version
         default: latest
         type: string
+      consolePluginImageURL:
```

Comment:
@didierofrivia  @maleck13 trying here complete URL instead of some "version". What are your thoughts about this?

Reply:
sorry guys, accepted by (answer) omission. 

We can always change that later